docs/configure.md

```diff
@@ -269,33 +269,14 @@ your fork is private **and** you are okay with storing secrets in it).
 
 ## Configure language servers
 
-> Code intelligence is a paid upgrade on top of the Data Center deployment option. After following these instructions to confirm it works, [buy code intelligence](https://about.sourcegraph.com/pricing).
-
-Code intelligence provides advanced code navigation and cross-references for your code on Sourcegraph.
-
-After setting up the base Sourcegraph deployment, refer to the following docs for each language for instructions about how to deploy / configure each language server:
-
-- [Go](../configure/xlang/go/README.md)
-- [Java](../configure/xlang/java/README.md)
-- [PHP](../configure/xlang/php/README.md)
-- [Python](../configure/xlang/python/README.md)
-- [Javascript / Typescript](../configure/xlang/typescript/README.md)
-
-We also have the following experimental language servers (please [read about the caveats](https://about.sourcegraph.com/docs/code-intelligence/experimental-language-servers/#caveats-of-experimental-language-servers) before enabling them):
-
-- [Bash](../configure/xlang/experimental/bash/README.md)
-- [Clojure](../configure/xlang/experimental/clojure/README.md)
-- [C++](../configure/xlang/experimental/cpp/README.md)
-- [C#](../configure/xlang/experimental/cs/README.md)
-- [CSS](../configure/xlang/experimental/css/README.md)
-- [Dockerfile](../configure/xlang/experimental/dockerfile/README.md)
-- [Elixir](../configure/xlang/experimental/elixir/README.md)
-- [HTML](../configure/xlang/experimental/html/README.md)
-- [Lua](../configure/xlang/experimental/lua/README.md)
-- [OCaml](../configure/xlang/experimental/ocaml/README.md)
-- [R](../configure/xlang/experimental/r/README.md)
-- [Ruby](../configure/xlang/experimental/ruby/README.md)
-- [Rust](../configure/xlang/experimental/rust/README.md)
+Code intelligence is provided through [Sourcegraph extensions](https://docs.sourcegraph.com/extensions). Refer to the READMEs for each language for instructions about how to deploy and configure them:
+
+- [Go](https://sourcegraph.com/extensions/chris/lang-go)
+- [JavaScript/TypeScript](https://sourcegraph.com/extensions/sourcegraph/lang-typescript)
+- [Python](https://sourcegraph.com/extensions/sourcegraph/python)
+- Java (coming soon)
```

Comment:
minor: Is there some other landing page that we can link to so we won't have to remember to update these links?

For example, link to a search on the extension registry that would include all current and future language servers?

Reply:
https://sourcegraph.com/extensions?query=lang works pretty well, but would miss `sourcegraph/python`

I'll at least replace `(coming soon)` with links to `?query=java`, for example.